tests/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ endif()
 find_file(DELPHES_CARD delphes_card_IDEA.tcl PATHS ${DELPHES_CARDS_DIR})
 if(DELPHES_CARD)
   if(BUILD_PYTHIA_READER)
-    ADD_COMPARISON_TEST(PythiaResonanceDecayFilter_pp_hhbbyy DelphesPythia8_EDM4HEP ${DELPHES_CARD} ${OUTPUT_CONFIG} pythia_resonance_decay_filter_pp_hhbbyy_output.root ${CMAKE_SOURCE_DIR}/examples/data/tester_pwp8_pp_hh_5f_hhbbyy.cmd)
+    ADD_COMPARISON_TEST(PythiaResonanceDecayFilter_pp_hhbbyy DelphesPythia8_EDM4HEP ${DELPHES_CARD} ${OUTPUT_CONFIG} pythia_resonance_decay_filter_pp_hhbbyy_output.root ${CMAKE_SOURCE_DIR}/examples/data/tester_pwp8_pp_hh_5f_hhbbyy.cmd --no-delphes)
```

Comment:
strictly speaking this is no longer a comparison test. Will have to clean this up at some point to make it less confusing.